neo4j/driver_with_context_examples_test.go

```diff
@@ -0,0 +1,82 @@
+package neo4j
+
+import (
+	"context"
+	"fmt"
+)
+
+var myDriver DriverWithContext
+var ctx context.Context
+
+func ExampleDriverWithContext_ExecuteQuery() {
```

Comment:
I don't fully understand how this test works, and what it's there for. I don't see any assertions of something.

Reply:
That's specific to the Go documentation toolchain: https://go.dev/blog/examples.
Examples will appear co-located to the docs.

Reply:
Nice eco-system feature!